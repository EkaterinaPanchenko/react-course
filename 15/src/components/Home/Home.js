import styled from 'styled-components';

const HomePage = styled.div`
    height: 50vh;
    display: flex;
    justify-content: center;
    align-items: center;
`;

function Home() {


    return(
        <HomePage>
            <h1>Age of Empires II</h1>
        </HomePage>
    );
};

export default Home;