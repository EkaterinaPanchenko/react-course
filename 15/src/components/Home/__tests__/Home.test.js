import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import Home from '../Home';

configure({ adapter: new Adapter() });

describe('<Home/> unit test', () => {
    const wrapper = shallow(<Home/>);

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});