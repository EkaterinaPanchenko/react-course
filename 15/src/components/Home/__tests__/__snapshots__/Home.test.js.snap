// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> unit test should match snapshot 1`] = `
<styled.div>
  <h1>
    Age of Empires II
  </h1>
</styled.div>
`;
